docs/visualize-data-with-grafana.rst

```diff
@@ -160,8 +161,8 @@ plot the number of rides per day in the first week of July 2019:
    Something important to know about the "Time series" format mode in Grafana
    is that your query needs to return a column called "time". Grafana will 
    identify this as your time metric, so make sure the column has the proper 
-   datatype (any datatype representing an `epoch time`_). In this query, we're 
-   labelling pickup_datetime as "time" for this reason.
+   datatype (any datatype representing an `epoch time`_). In this query, 
+   we're labelling pickup_datetime as "time" for this reason.
```

Comment:
maybe you should use the American spelling of "labeling" here too?

Reply:
Absolutely, oversight on my side. Thanks!